map_app/map_app/DataManager.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  DataManager.swift
+//  map_app
+//
+//  Created by Allison Collins on 2/12/20.
+//  Copyright © 2020 Allison Collins. All rights reserved.
+//
+
+import Foundation
+import MapKit
+let defaults = UserDefaults.standard
+
+public class DataManager {
+    //Mark: - Singleton Stuff
+    public static let sharedInstance = DataManager()
+    
+    //this prevents others from using the default '()'
+    fileprivate init() {
+        self.loadAnnotationFromPlist()
+        defaults.set([Place](), forKey: "favePlaces")
+    }
+    
+    struct ListData : Codable {
+        var places : [aPlace]?
+    }
+    
+    struct aPlace: Codable {
```

Comment:
This naming convention is not proper. Stick to defining structs and classes both with capital letters.